src/main/java/com/example/bookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,21 @@
+package com.example.bookstore.dto.user;
+
+import com.example.bookstore.lib.PasswordValidator;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserLoginRequestDto {
+    @Size(min = 7, max = 25)
+    @Email
+    private String email;
+
+    @PasswordValidator
+    private String password;
```

Comment:
you do not have PasswordValidator in UserRegistrationRequestDto 
So, user can register with a pure password and then... )))